packages/next-bundle-analyzer/index.js

```diff
@@ -0,0 +1,28 @@
+module.exports = (nextConfig = {}) => {
+  return Object.assign({}, nextConfig, {
+    webpack (config, options) {
+      const { isServer } = options
+      const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer')
+
+      if (nextConfig.analyze) {
+        config.plugins.push(
+          new BundleAnalyzerPlugin(
+            Object.assign(
```

Comment:
There's no need for Object.assign here if we don't merge anything else.

Let's use this config:

```
analyzerMode: 'static',
reportFilename: options.isServer ? '../analyze/server.html' : './analyze/client.html'
```

Reply:
Updated